corehq/util/pagination.py

```diff
@@ -154,7 +166,7 @@ def __init__(self, iterator_state, args_provider):
 
     def get_initial_args(self):
         if self.resume:
-            return self.resume_args, self.resume_kwargs
+            return unpack_jsonobject(self.resume_args), unpack_jsonobject(self.resume_kwargs)
```

Comment:
I think this probably isn't used a ton, but any ideas on how much overhead this will add?

Reply:
I suspect the overhead is small, since these are just ```ResumableIteratorState```s, which don't have a lot of depth or data: https://github.com/dimagi/commcare-hq/blob/552c6cfc31718c9558cdebef183212fc726fedee/corehq/util/pagination.py#L136-L145